document/core/exec/runtime.rst

```diff
@@ -26,7 +26,8 @@ It is convenient to reuse the same notation as for the |CONST| :ref:`instruction
      \I32.\CONST~\i32 \\&&|&
      \I64.\CONST~\i64 \\&&|&
      \F32.\CONST~\f32 \\&&|&
-     \F64.\CONST~\f64
+     \F64.\CONST~\f64 \\&&|&
+     \S128.\CONST~\s128
```

Comment:
Unfortunately, you cannot use `s128` as the meta variable for SIMD constants, since that already is defined to be a signed 128-bit integer. However, if you represent SIMD values as i128 (see comment below) you could use that here.